src/kernel/include/shared/cos_types.h

```diff
@@ -15,7 +15,7 @@
 #define TYPES_H
 
 #include "./consts.h"
-#include "./cos_config.h"
+#include "../chal/shared/cos_config.h"
```

Comment:
Should `cos_types.h` (or some subset of it) also be in the `chal`?

Reply:
I guess so. These will be moved in next pr